variables.tf

```diff
@@ -225,7 +227,7 @@ variable "create_jump_vm" {
 }
 
 variable "create_jump_public_ip" {
-  default = null
+  default = false
```

Comment:
This one is usually `true`, by default we should always stand up infra that can work, test with the [defaults](https://github.com/sassoftware/viya4-iac-azure/blob/main/examples/sample-input-defaults.tfvars) example file.

Reply:
fixed and tested.